lib/code_corps/policy/policy.ex

```diff
@@ -24,6 +24,10 @@ defmodule CodeCorps.Policy do
   end
 
   @spec can?(User.t, atom, struct, map) :: boolean
+  defp can?(%User{} = user, :create, %Category{}, %{}), do: Policy.Category.create?(user)
+  defp can?(%User{} = user, :update, %Category{}, %{}), do: Policy.Category.update?(user)
+
+  @spec can?(User.t, atom, struct, map) :: boolean
```

Comment:
These all have the same "prototype" (same amount of arguments, same types of arguments in the same order), so there is no need for an extra `@spec`, nor for keeping a blank line between them. Elixir, and Credo prefer no blank lines between the same function's (in the context of arity) different clauses.

This is something that wasn't clear since there was only a single example in the code previously, but should be clearer once we merge your PR in.